File: test/fixtures/php/j_post_form_f.php
<?php
$ch = curl_init();
curl_setopt($ch, CURLOPT_URL, 'http://localhost:28139/post');
curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
curl_setopt($ch, CURLOPT_CUSTOMREQUEST, 'POST');
curl_setopt($ch, CURLOPT_POSTFIELDS, [
    'd1' => 'data1',
    'd2' => 'data',
]);

$response = curl_exec($ch);

curl_close($ch);

File: test/fixtures/dart/j_post_form_f.dart
import 'package:http/http.dart' as http;

void main() async {
  final url = Uri.parse('http://localhost:28139/post');

  final req = http.MultipartRequest('POST', url)
    ..fields['d1'] = 'data1'
    ..fields['d2'] = 'data';

  final stream = await req.send();
  final res = await http.Response.fromStream(stream);
  final status = res.statusCode;
  if (status != 200) throw Exception('http.send error: statusCode= $status');

  print(res.body);
}
